app/models/season/phase_switcher.rb

```diff
@@ -50,4 +61,12 @@ def self.back_to_reality
     })
   end
 
+  def self.whitelisted_phases
```

Comment:
Just outta curiosity @F3PiX, why did you chose a memorized class method over a constant?
I'd intuitively go with
```ruby
PHASES = %i(fake_proposal_phase ...)
```
Also I prefer symbols over strings 😉